deploy/contents/install/app/install-utils.sh

```diff
@@ -562,6 +562,41 @@ function array_contains_or_empty () {
     return 1
 }
 
+function count_services {
+    parse_env_option "$1"
+    services_count=$((services_count+1))
+    if [ -n "${CP_SERVICES_ENABLED}" ]; then 
+        export CP_SERVICES_ENABLED="${CP_SERVICES_ENABLED},${1}"
+    else 
+        export CP_SERVICES_ENABLED="${1}"
+    fi
+    }
+
+function export_service_all {
+        export CP_INSTALL_SERVICES_ALL=1
+        export CP_SERVICES_ENABLED="all"
+    }
+
+function services_from_pitc {
+    local services_list="$CP_POINT_IN_TIME_CONFIGURATION_DIR/cp-services.json"
+    if [ ! -s "$services_list" ]; then
+       export_service_all
+    else
+       while IFS=',' read -r app switch; do
```

Comment:
I propose to move code that check that we have all necessary files for this step to a different function. Then here we can use this function.
It would be even better and natural to make such function common to be able to check presents of any configuration in `pitc` folder, based on passed parameter